packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -342,24 +384,206 @@ describe('GCFBootstrapper', () => {
+        sinon.assert.notCalled(issueSpy);
+        sinon.assert.notCalled(repositoryCronSpy);
+        sinon.assert.notCalled(installationCronSpy);
+        sinon.assert.notCalled(globalCronSpy);
+        listInstallationRequests.done();
+      });
+
+      it('ensures that task is enqueued when called by scheduler for many installations', async () => {
+        await mockBootstrapper();
+        req.body = {
+          cron_type: 'installation',
+        };
+        req.headers = {};
+        req.headers['x-github-event'] = 'schedule.repository';
+        req.headers['x-github-delivery'] = '123';
+        req.headers['x-cloudtasks-taskname'] = '';
+        const listInstallationRequests = nockListInstallations(
+          'app_installations_multiple.json'
+        );
+
+        await handler(req, response);
+
+        sinon.assert.calledTwice(enqueueTask);
+        sinon.assert.notCalled(issueSpy);
+        sinon.assert.notCalled(repositoryCronSpy);
+        sinon.assert.notCalled(installationCronSpy);
+        sinon.assert.notCalled(globalCronSpy);
+        listInstallationRequests.done();
+      });
+
+      it('ensures that task is enqueued when called by scheduler with an installation id', async () => {
+        await mockBootstrapper();
+        req.body = {
+          cron_type: 'installation',
+          installation: {
+            id: 1,
+          },
+        };
+        req.headers = {};
+        req.headers['x-github-event'] = 'schedule.repository';
+        req.headers['x-github-delivery'] = '123';
+        req.headers['x-cloudtasks-taskname'] = '';
+
+        await handler(req, response);
+
+        sinon.assert.calledOnce(enqueueTask);
+        sinon.assert.notCalled(issueSpy);
+        sinon.assert.notCalled(repositoryCronSpy);
+        sinon.assert.notCalled(installationCronSpy);
+        sinon.assert.notCalled(globalCronSpy);
+      });
+
+      it('handles the schedule.installation task', async () => {
+        await mockBootstrapper();
+        req.body = {
+          installation: {id: 1},
+          cron_type: 'installation',
+          cron_org: 'some-cron-org',
+        };
+        req.headers = {};
+        req.headers['x-github-event'] = 'schedule.installation';
+        req.headers['x-github-delivery'] = '123';
+        req.headers['x-cloudtasks-taskname'] = 'test-function';
+
+        await handler(req, response);
+
+        sinon.assert.notCalled(enqueueTask);
+        sinon.assert.notCalled(issueSpy);
+        sinon.assert.notCalled(repositoryCronSpy);
+        sinon.assert.calledOnce(installationCronSpy);
+        sinon.assert.notCalled(globalCronSpy);
+      });
+    });
+
+    describe('global cron', () => {
+      it('enqueues a single task for global scheduled task', async () => {
+        await mockBootstrapper();
+        req.body = {
+          cron_type: 'global',
+        };
+        req.headers = {};
+        req.headers['x-github-event'] = 'schedule.repository';
```

Comment:
`schedule.global` instead of `schedule.repository`?

Reply:
same as above